src/components/editor/editor-component.ts

```diff
@@ -287,10 +288,23 @@ export function view(
         updateMetadata,
         deleteNode,
       ),
-
       html.div(
         {
           id: 'editor',
+          onclick: (ev: Event) => {
```

Comment:
Ideally you'd try and keep the view function as logic free as possible and move out complex logic into an actions.

It's fine for now, don't worry about it right now.